README.md

```diff
@@ -166,11 +168,11 @@ This is my way of giving back to the QEMU, KVM, and VFIO community. Please don't
 
 - I'm not responsible for any time you waste using this project.
 - I'm not responsible if you make an oopsie whoopsie.
-- Do NOT run anything as ``sudo``. If you have an urge to then you need to break that habit _**asap.**_ None of my scripts require superuser privileges.
+- Do NOT run anything as ``sudo``. If you have an urge to then you need to break that habit _**asap.**_ Most of my scripts do not require superuser privileges, however, the ones that do require them are mentioned.
```

Comment:
Needs work; change to "the ones that do are clearly marked with a yellow ⚠️ next to the operation requiring such permissions."